ztp/gitops-subscriptions/argocd/deployment/sites-app.yaml

```diff
@@ -16,7 +16,7 @@ spec:
   source:
     directory:
       recurse: true
-    path: ztp/ztp-policy-generator/out/sites
+    path: ztp/gitops-subscriptions/argocd/resource-hook-example
```

Comment:
You mixing clusters-app with sites-app. Clusters-app for siteconfigs AND Sites-app for sites specific policies

Reply:
fixed